spec/features/bulk_updates/bulk_updates_location_keywords_spec.rb

```diff
@@ -186,12 +184,11 @@
         before do
           visit collection_path(@add_to_existing_ingest_response['concept-id'])
 
-          find('.tab-label', text: 'Additional Information').click
         end
 
         it 'displays the new keyword' do
-          within '.location-keywords-preview' do
-            expect(page).to have_content('OCEAN ATLANTIC OCEAN NORTH ATLANTIC OCEAN BALTIC SEA', normalize_ws: true)
+          within '#metadata-preview' do
+            expect(page).to have_no_content('OCEAN ATLANTIC OCEAN NORTH ATLANTIC OCEAN BALTIC SEA', normalize_ws: true)
```

Comment:
You will need to split these up and look for individual phrases, like OCEAN, ATLANTIC OCEAN, NORTH ATLANTIC OCEAN, BALTIC SEA.   Put in each into there own expect, and it needs to say "have_content"